modules/Vtiger/uitypes/String.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * String UIType field file.
+ *
+ * @package   UIType
+ *
+ * @copyright YetiForce S.A.
+ * @license   YetiForce Public License 5.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Adrian Kon <a.kon@yetiforce.com>
+ */
+
+/**
+ * String UIType field class.
+ */
+class Vtiger_String_UIType extends Vtiger_Base_UIType
+{
+	/** {@inheritdoc} */
+	protected $isResizableColumn = true;
+
+	/** {@inheritdoc} */
+	public function validateMaximumLength(int $minimumLength, int $maximumLength): bool
+	{
+		$allowedMaxValue = 255;
+		$newColumnLength = (int) $maximumLength;
+		return $newColumnLength <= $allowedMaxValue;
+	}
```

Comment:
you have something like this here: https://github.com/YetiForceCompany/YetiForceCRM/pull/17204/files#diff-70152fd4caea113f66f8f617348ab9eedf6d5f22bd6e7ea23a64d558211b6374R1713-R1716
and in the other uitype you use: https://github.com/YetiForceCompany/YetiForceCRM/pull/17204/files#diff-817852a30f3e1bf77315f8c697104e60cc94ad9a3b1448d2f231ea81e317cfe1R95-R99
why you can't use the same here?